plugins/content_distribution/providers/youtube_api/lib/YoutubeApiDistributionEngine.php

```diff
@@ -682,5 +687,25 @@ private static function uploadChunk($media, $chunk)
 
 	}
 
+	/**
+	 * @param int $fileSize
+	 */
+	protected function setLongProcessedTimeout($fileSize)
+	{
+		if (isset(KBatchBase::$taskConfig->params->youtubeApi))
+		{
+			if (isset(KBatchBase::$taskConfig->params->youtubeApi->longProcessedTimeout))
+				$this->longProcessedTimeout = KBatchBase::$taskConfig->params->youtubeApi->longProcessedTimeout;
```

Comment:
If value is not set in the configuration file - supply default value hard coded

Reply:
default values set lines 33-35